io.github.amourspirit.LibrePythonista_PyEditor.yml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.amourspirit.LibrePythonista_PyEditor
+runtime: org.gnome.Platform
+runtime-version: '47'
```

Comment:
why does it need gnome runtime?

Reply:
> I asked copilot 

DON'T. You waste everyone time with that kind of things.

Reply:
OK, What is the proper solution for runtime? 
This is my first flatpak so I am not sure what proper runtime should be.